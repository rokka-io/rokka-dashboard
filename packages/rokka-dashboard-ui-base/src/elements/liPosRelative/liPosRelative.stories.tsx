import { storiesOf } from '@storybook/react';
import React from 'react';
import { colors } from '../../identity/colors/colors';
import { LiPosRelative } from './liPosRelative';

storiesOf('Elements / Sidebar', module)
  .addDecorator(story => <div style={{ background: colors.brand.dark }}>{story()}</div>)
  .add('LiPosRelative', () => (
    <ul>
      <LiPosRelative>Li pos relative</LiPosRelative>
    </ul>
  ));
