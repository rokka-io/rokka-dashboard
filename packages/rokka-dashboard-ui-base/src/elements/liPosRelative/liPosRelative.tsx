import styled from 'styled-components';

export const LiPosRelative = styled.li`
  position: relative;
`;
