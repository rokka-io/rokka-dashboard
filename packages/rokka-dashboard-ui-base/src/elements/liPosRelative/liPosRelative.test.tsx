import { shallow } from 'enzyme';
import React from 'react';
import { LiPosRelative } from './liPosRelative';

describe('Elements', () => {
  describe('Sidebar', () => {
    it('renders a LiPosRelative correctly.', () => {
      const component = shallow(<LiPosRelative />);

      expect(component).toMatchSnapshot();
    });
  });
});
