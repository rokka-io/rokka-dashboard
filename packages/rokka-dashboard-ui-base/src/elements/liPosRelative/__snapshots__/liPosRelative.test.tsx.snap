// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Elements Sidebar renders a LiPosRelative correctly. 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  position: relative;
",
        ],
      },
      "displayName": "styled.li",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "li",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
